src/index.ts

```diff
@@ -29,10 +29,19 @@ export default function prevalPlugin(
 
         prevalComment.value = ' this file was prevaled'
 
-        const result = transformFromAstSync(path.node, undefined, fileOpts)
-
-        // istanbul ignore next because this should never happen, but TypeScript needs me to handle it
-        const string = result?.code ?? ''
+        // @ts-expect-error the types for this is wrong...
+        const {code: string} = transformFromAst(
+          path.node,
+          // @ts-expect-error the types for this is wrong...
+          null,
+          /* istanbul ignore next (babel 6 vs babel 7 check) */
+          babel.version.startsWith('6.')
```

Comment:
Note, I did change this from an `eslint` warning from `/^6\./.test(babel.version)`